src/lib/util/Config.ts

```diff
@@ -19,26 +20,26 @@ export type EngineConfigContent = {
 }
 
 const CONFIG_FILE_PATH = path.join(SFDX_SCANNER_PATH, CONFIG_FILE);
-const DEFAULT_CONFIG: ConfigContent = {
+export const DEFAULT_CONFIG: ConfigContent = {
```

Comment:
Would this be a good opportunity to switch this to a Map? It would simplify lookups an prevent duplicates.

Reply:
In Scanner 1.0, Config was a Map. This became a JSON as a part of Mike's initial changes to support multiple engines - I suppose there's an aspect that a Map couldn't support. Let me spend sometime to understand this better and see if we can modify Config structure before Scanner2.0 is shipped.